src/api/track.ts

```diff
@@ -14,6 +10,35 @@ import TripsAPI from './trips';
 
 import type { RadarTrackParams, RadarTrackResponse } from '../types';
 
+const base64Encode = (str: string): string => 
+  btoa(str).replace(/\+/g, '-').replace(/\//g, '_').replace(/=+$/, '');
+
+const signJWT = async (payload: object, key: string): Promise<string> => {
```

Comment:
nit: can move this function to a util file: `/src/util/jwt.ts`

Reply:
Done